pkg/controller.v1/pytorch/pytorchjob_controller.go

```diff
@@ -205,43 +196,49 @@ func (r *PyTorchJobReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 
 // SetupWithManager sets up the controller with the Manager.
 func (r *PyTorchJobReconciler) SetupWithManager(mgr ctrl.Manager) error {
```

Comment:
Can you explain the changes here?

Reply:
I found the logging shows `onOwnerCreateFunc` in pytorch is called when pods&services created by tf-controller. I double checked XGBoost part and found such functions are not used. Getting rid of these setting up does not affect functionality of pytorch controller.